scripts/us_census/acs5yr/subject_tables/common/data_loader.py

```diff
@@ -106,6 +103,7 @@ def process_subject_tables(table_prefix='',
                                              header_row=header_row)
 
     ## if input_path is a file, select csv/zip processing methods
+    input_path = os.path.expanduser(input_path)
```

Comment:
Is this a necessary change? 
`os.path.expanuser()` method is only useful if the paths are specified relative to a home directory. In this case, why do we need this extra step?
Applies to all the lines where `os.path.expanduser()` is proposed.

Reply:
I do use it that way.

Reply:
Ok, then please have this change in `process.py` since that serves as the wrapper to these modules.

Reply:
done!